troposphere/static/css/app/instance.scss

```diff
@@ -25,6 +25,10 @@
   &.inactive {
     background: crimson;
   }
+
+  &.error {
+    background: crimson;
+  }
```

Comment:
So the default is grey (no class attached), im not sure why inactive is crimson..